front/README.md

```diff
@@ -40,26 +40,35 @@ Launches end to end tests.
 It requires:
 
 - Install playwright dependencies `cd ./front/ && yarn playwright install --with-deps`
-- Backend containers to be up: `docker compose up --no-build --detach valkey postgres gateway core editoast`
+- Backend containers to be up:
+  `docker compose up --no-build --detach valkey postgres gateway core editoast`
 - Running front with `docker compose up --build --detach front`
 
 Now you can run the test with `cd front/ && yarn e2e-tests`.
 
-> [!CAUTION]
-> If you try to run `yarn start` instead of running it through docker, you'll notice it doesn't
-> work because the gateway can't access your local port from inside a container. 2 solutions:
+If your os is not supported by playwright (which only supports windows, macos and ubuntu/debian),
+you can instead start the tests inside a docker container using
```

Comment:
That is, if your OS isn't supported and is Linux.

Reply:
Sure, sure. But are there many other OS we care to support for dev purposes? ^^''

Reply:
We don't really, so let's not pretend we do ;)

Reply:
Done, thanks!